acceptance/config/masterless-windows-2008-x86_64.cfg

```diff
@@ -1,17 +0,0 @@
-HOSTS:
```

Comment:
Can we remove these old configs in a separate commit?

Are they used by CI?

Reply:
None of the nodeset is used in CI. All generated on the fly with BHG.